### ISSUE TITLE
Failed at the angular2-quickstart@1.0.0 lite script 'lite-server'

### ISSUE THREAD

**Issue by sadikr** [johnpapa/lite-server#54]
** Possible duplicate of  [https://github.com/johnpapa/lite-server/issues/49](url)

System details : 
Ubuntu 14.04 LTS
Node v5.0.0
npm v3.0.0

See the attached npm-debug.log.
[npm-debug.txt](https://github.com/johnpapa/lite-server/files/177593/npm-debug.txt)

**Comment by johnpapa** [comment]
I'm unable to reproduce. Have you tried clearing NPM cache, deleting your node_modiles,  and trying again?

**Comment by bruceherve** [comment]
I have similar system details to yours. I had the issue but managed to resolve it by:
 clear the npm cache, then 
 npm install -g concurrently lite-server typescript 
 delete the node_modules folder 
then do npm install again

**Comment by sadikr** [comment]
@bruceherve @johnpapa  I tried your suggestions but my bad. While installing 'lite-server', it stuck with 
**bufferutil@1.2.1 install: `node-gyp rebuild`**
Seems like issue is not with lite-server. [https://github.com/websockets/bufferutil/issues/36](url)

Just for the information sake, can you please mention your bufferutil version?

**Comment by bruceherve** [comment]
@sadikr.  the bufferutil is 1.2.1, just like yours.. may I suggest that you reinstall node-gyp globally as well. Because some packages do try to recompile by using node-gyp, most of the time you only get a warning though. Try that and see If it helps

**Comment by bruceherve** [comment]
I am using node V5.3 by the way, just in case my latest suggestion does not work.. because I see that OS(ubuntu 14 LTS) wise it's the same apart from the specific node(5.x) version which in my case is 5.3 , So you might want to try that as well.

**Comment by sadikr** [comment]
@bruceherve : no luck.

**Comment by bruceherve** [comment]
@sadikr. Also do npm install -g typings and  @johnpapa  suggests to increases the number of available File Watches by running the following command: 
echo fs.inotify.max_user_watches=524288 | sudo tee -a /etc/sysctl.conf && sudo sysctl -p
It did help people on ubuntu 15.10..
But mainly reinstall lite-server:
npm install -g lite-server

**Comment by johnpapa** [comment]
closing this as a dupe. move there so we can track npm issues there

**Comment by johnpapa** [comment]
actually let's use this one as the trail. reopening

**Comment by johnpapa** [comment]
The recommendation is to use lite-server as a local, but yes it can be used globally too.

**Comment by cgmartin** [comment]
I am not able to reproduce with Ubuntu 14.04, nodejs 5.9.0, npm 3.7.3. I don't believe this is lite-server specific, but a problem with your nodejs environment.

Are `build-essential` and `python` packages installed?

The commands in the following Dockerfile are what I ran on a fresh Ubuntu 14.04 image, with lite-server running successfully:
```Dockerfile
# Dockerfile
FROM ubuntu:14.04

EXPOSE 3000 3001

RUN apt-get update && \
    apt-get install -y curl build-essential python && \
    (curl -sL https://deb.nodesource.com/setup_5.x | sudo -E bash -) && \
    apt-get install -y nodejs && \
    mkdir -p /html && \
    (echo "<!DOCTYPE html><h1>Lite-Server Test</h1>" > /html/index.html) && \
    npm i -g lite-server

WORKDIR /html
VOLUME /html

CMD lite-server
```

To build/run:
```
$ docker build -t lite-server-ubuntu .
$ docker run -it --rm -p 3000:3000 -p 3001:3001 lite-server-ubuntu
$ open http://$(docker-machine ip):3000
```

**Comment by cgmartin** [comment]
Likely fixed with #57. Closing.
